src/components/Review.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import { Paper } from '@material-ui/core';
+
+const StyledTableCell = withStyles((theme) => ({
+  head: {
+    backgroundColor: theme.palette.common.black,
+    color: theme.palette.common.white,
+  },
+  body: {
+    fontSize: 14,
+  },
+}))(TableCell);
+
+const StyledTableRow = withStyles((theme) => ({
+  root: {
+    '&:nth-of-type(odd)': {
+      backgroundColor: theme.palette.action.hover,
+    },
+  },
+}))(TableRow);
+
+function createData(name, Excerpt, More) {
+  return { name, Excerpt, More };
+}
+
+const rows = [
+  createData('Nomy', "it was good", 6.0),
+  createData('Warrier', "bad experience", 9.0),
+  createData('Elaxender', "Too good for society", 16.0),
+  createData('pupchu', "not for students", 3.7),
+  createData('enderson', "Good", 16.0),
+];
+
+const useStyles = makeStyles({
+  table: {
+    minWidth: 700,
+  },
+});
+
+export default function Basic() {
```

Comment:
Use meaningful names, this component is for Reviews, it's name should be similar to reviews

Reply:
and use class-based component